src/components/Characters/GenderFlower.js

```diff
@@ -1,35 +1,55 @@
-      />
-    </svg>
-  );
-}
-
-const GenderFlower = styled(GenderFlowerArtwork)`
+const GenderFlowerStyled = styled(animated.span)`
   position: absolute;
   z-index: ${zIndexArtworkMainDecor};
   color: ${props => props.theme.primaryColor};
+  transform-origin: '50% 50%';
 `;
 
+function GenderFlower(props) {
+  const { x } = useSpring({
```

Comment:
Maybe you can try something like that:

```
  const props = useSpring({
    from: { transform: 'scale(0) rotate(0deg)' },
    to: { transform: 'scale(1) rotate(360deg)' }
    config: config.wobbly
  });
```

Also if you want to add an animation also for the `unmount` of component, just use `useTransition`

Reply:
Or smth like that:
```
  const { scale, rotation } = useSpring({
    from: { scale: 0, rotation: 0 },
    to: { scale: 1, rotation: 360 }
    config: config.wobbly
  });

return <animated.flower style={{ transform: `scale(${scale}) rotation(${rotation})` }} />
```

Reply:
>I think that we shouldn't use interpolation for such simple animations. 

I guess using `interpolation` is the only way to create _custom_ keyframes based animation, right? That's why I did it, to get slightly more appealing animation than _linear_ transition like you've suggested. 

```
    from: { scale: 0, rotation: 0 },
    to: { scale: 1, rotation: 360 }
```
On short animation it's harder to notice... Anyway,  I've tried ` config: config.wobbly` and using this predefined config animation looks even better 😎
Fixed in: 3af19aae897caccb437ea724e1114c1cbf1368e2